src/components/orders/orders_table.vue

```diff
@@ -332,75 +331,76 @@
 			}
 
 		},
+
 		mounted() {
 			initStages(this.stages);
 			initDB(this, fetchAndDisplayOrders);
 		},
 	};
 
-	const stages: string[] = ["接单", "订货", "厂家接单", "我收到", "飘洋过海", "寄出", "买家收到(完成)"];
+	const stages = ["接单", "订货", "厂家接单", "我收到", "飘洋过海", "寄出", "买家收到(完成)"];
 
-	function initStages(obj) {
+	const initStages = (obj) => {
 		obj = stages;
 	}
 
 	class HistoryItem {
 		stage: number
 		date: Date
 
-		public constructor(id: number, date_str: string) {
+		public constructor(id: number, dateStr: string) {
 			this.stage = id;
-			this.date = new Date(date_str);
+			this.date = new Date(dateStr);
 		}
 	}
 
 	class Order {
 		id: number
 		user: string
 		bag: string
-		buy_price: number
-		sell_price: number
+		buyPrice: number
+		sellPrice: number
 		send_price: number
 		sender: string
 		benefit: number
 		stage: number
-		stage_str: string
+		stageStr: string
 		update: Date
-		update_str: string
-		history: Array < HistoryItem > = []
+		updateStr: string
+		history: Array<HistoryItem> = []
 
-		public constructor(order: any) {
+		public constructor(order: Order) {
 			this.id = order.id;
 			this.user = order.user;
 			this.bag = order.bag;
-			this.buy_price = order.buy_price;
-			this.sell_price = order.sell_price;
+			this.buyPrice = order.buyPrice;
+			this.sellPrice = order.sellPrice;
```

Comment:
This param in other file also need to be change. I will follow this format to change all parameters.